src/grandorgue/GOFrame.cpp

```diff
@@ -545,6 +545,13 @@ void GOFrame::Init(wxString filename) {
   clean.Cleanup();
 }
 
+void GOFrame::AttachToOrganController(bool isToAttach) {
```

Comment:
Just wondering why calling a function that attachs with a parameter like that. Either the function name means something else, either the parameter does.

Reply:
Maybe a comment should be welcome

Reply:
Renamed to AttachDetachOrganController